system_setup/cmd/tui.py

```diff
@@ -48,28 +48,16 @@ def make_client_args_parser():
     parser.add_argument('--dry-run', action='store_true',
                         dest='dry_run',
                         help='menu-only, do not call installer function')
-    # TODO WSL: remove any uneeded arguments
     parser.add_argument('--socket')
     parser.add_argument('--serial', action='store_true',
```

Comment:
Can you give me some examples on how --serial is used under WSL? (Like a pragmatical step by step guide). I think if this work, we should document and see how this can be useful for users.